### PR TITLE
Adds bounty page to docs site

### PR DESCRIPTION
**Pull request by 10thfloor**
This PR adds 2 documents: 

- A landing page with information about our bug-bounty program
- Responsible Disclosure Policy

### CODE REVIEW

File: docs/content/bounties/index.mdx [hunk 1]
```diff
@@ -0,0 +1,102 @@
+---
+title: Flow Bug Bounty Program
+description: Earn up to 100k by finding Flow bugs!
+sidebar_title: Bug Bounties
+---
+
+<div>
+  <Button feel="flat" size="large">
```
Comment (turbolent) [inline docs/content/bounties/index.mdx:8]: This button doesn't seem to do anything

File: docs/content/bounties/responsible-disclosure.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+## Disclosure Acknowledgements
+  
+  Security acknowledgements can be found at https://dapperlabs.com/security_nods.txt
+
+## Reporting Security Findings
+
+Reports welcome! Please do reach out to us if you have a security concern. If you believe you may have found a security vulnerability in one of our products or platforms, send us an email: [security@dapperlabs.com](mailto:security@dapperlabs.com?subject=Responsible%20disclosure)
+
+We encourage you to encrypt the information you send us using our PGP key at https://dapperlabs.com/security@dapperlabs.com.pgp.txt
```
Comment (turbolent) [inline docs/content/bounties/responsible-disclosure.md:76]: Not sure where this is from, but the email and key should be the ones as found here: 
https://github.com/onflow/cadence/blob/master/SECURITY.md#reportingsecurityfindingstous

**Review by DapperDrew — APPROVED** [review]

File: docs/content/bounties/index.mdx [hunk 3]
```diff
@@ -0,0 +1,102 @@
+---
+title: Flow Bug Bounty Program
+description: Earn up to 100k by finding Flow bugs!
```
Comment (brianastrove) [inline docs/content/bounties/index.mdx:3]: "Earn 100k or more by finding ...."

Reward for critical is minimum 100k

File: docs/content/bounties/index.mdx [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+<a href="/bounties/responsible-disclosure">
+  Read our Responsible Disclosure Guidelines
+</a>
+
+## Rewards
+
+<p>
+  We run closed bug bounty programs, but beyond that, we also pay out rewards
+  once per eligible bug to the first responsibly disclosing third party.
+  Rewards are based on the seriousness of the bug, but the minimum is $100 and
```
Comment (brianastrove) [inline docs/content/bounties/index.mdx:21]: min = $2,500 USD value

File: docs/content/bounties/index.mdx [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+  we have and are willing to pay $100,000 or more at our sole discretion.
```
Comment (brianastrove) [inline docs/content/bounties/index.mdx:22]: remove "we have" b/c i dont think we have yet, but we are willing to pay 100K+

File: docs/content/bounties/index.mdx (hunk 3 above)
Comment (brianastrove) [reply docs/content/bounties/index.mdx:3]: "Earn $100k USD value by finding Flow bugs" would be better too b/c can get rewards in different currencies.

File: docs/content/bounties/index.mdx (hunk 1 above)
Comment (10thfloor) [reply docs/content/bounties/index.mdx:8]: Not hooked up for security reasons. Will connect when approved.

File: docs/content/bounties/responsible-disclosure.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
+---
+title: Responsible Disclosure Policy
+description: Policy and guidelines for submitting security related bugs.
+---
+
+<div>
+  <button>Submit a Bug</button>
```
[the remark this answers is not in the document]
Comment (10thfloor) [reply docs/content/bounties/responsible-disclosure.md:7]: See above coment.

File: docs/content/bounties/index.mdx [hunk 7]
```diff
@@ -16,8 +16,8 @@ sidebar_title: Bug Bounties
 ## Rewards
 
 <p>
-  We run closed bug bounty programs, but beyond that we also pay out rewards,
-  once per eligible bug, to the first responsibly disclosing third party.
+  We run closed bug bounty programs, but beyond that, we also pay out rewards
+  once per eligible bug to the first responsibly disclosing third party.
   Rewards are based on the seriousness of the bug, but the minimum is $100 and
```
Comment (10thfloor) [inline docs/content/bounties/index.mdx:21]:
```suggestion
  Rewards are based on the seriousness of the bug, but the minimum is $2,500 USD and
```

File: docs/content/bounties/index.mdx [hunk 8]
```diff
@@ -16,8 +16,8 @@ sidebar_title: Bug Bounties
...
   we have and are willing to pay $100,000 or more at our sole discretion.
```
Comment (10thfloor) [inline docs/content/bounties/index.mdx:22]:
```suggestion
  we are willing to pay $100,000 or more at our sole discretion.
```

File: docs/content/bounties/index.mdx (hunk 3 above)
Comment (10thfloor) [inline docs/content/bounties/index.mdx:3]:
```suggestion
description: Earn $100k USD by finding Flow bugs
```

File: docs/content/bounties/responsible-disclosure.md [hunk 9]
```diff
@@ -0,0 +1,84 @@
...
+- At a minimum, we will recognize your contribution in our Disclosure Acknowledgements if you are the first to report the issue and we make a code or configuration change based on the issue.
+
+## Disclosure Acknowledgements
+  
+  Security acknowledgements can be found at https://dapperlabs.com/security_nods.txt
+
+## Reporting Security Findings
+
+Reports welcome! Please do reach out to us if you have a security concern. If you believe you may have found a security vulnerability in one of our products or platforms, send us an email: [security@dapperlabs.com](mailto:security@dapperlabs.com?subject=Responsible%20disclosure)
+
```
Comment (10thfloor) [inline docs/content/bounties/responsible-disclosure.md:75]:
```suggestion
Reports welcome! Please do reach out to us if you have a security concern. If you believe you may have found a security vulnerability in one of our products or platforms, send us an email: [security@dapperlabs.com](mailto:security@onflow.org?subject=Responsible%20disclosure)

```

File: docs/content/bounties/responsible-disclosure.md (hunk 2 above)
Comment (10thfloor) [inline docs/content/bounties/responsible-disclosure.md:76]:
```suggestion
We encourage you to encrypt the information you send us using our PGP key at [keys.openpgp.org/security@onflow.org](keys.openpgp.org/security@onflow.org)


```

File: docs/content/bounties/responsible-disclosure.md [hunk 10]
```diff
@@ -0,0 +1,86 @@
...
+We encourage you to encrypt the information you send us using our PGP key at [keys.openpgp.org/security@onflow.org](keys.openpgp.org/security@onflow.org)
+
+
+
+[🔐 PGP Public Key](https://www.notion.so/PGP-Public-Key-dc945aa551904be0a8d0a48b44e6bf68)
```
Comment (10thfloor) [inline docs/content/bounties/responsible-disclosure.md:80]:
```suggestion

```

File: docs/content/bounties/index.mdx (hunk 3 above)
Comment (10thfloor) [inline docs/content/bounties/index.mdx]:
```suggestion
description: Earn $100k USD by finding Flow bugs
```

File: docs/content/bounties/index.mdx [hunk 11]
```diff
@@ -0,0 +1,102 @@
...
+title: Flow Bug Bounty Program
+description: Earn $100k USD by finding Flow bugs
+sidebar_title: Bug Bounties
+---
+
+<div>
+  <Button feel="flat" size="large">
+    Submit a Bug
+  </Button>
+</div>
```
Comment (10thfloor) [inline docs/content/bounties/index.mdx:11]:
```suggestion
If you believe you may have found a security vulnerability in one of our products or platforms, send us an email: [security@dapperlabs.com](mailto:security@onflow.org?subject=Responsible%20disclosure)
```

File: docs/content/bounties/responsible-disclosure.md [hunk 12]
```diff
@@ -0,0 +1,85 @@
+---
+title: Responsible Disclosure Policy
+description: Policy and guidelines for submitting security related bugs.
+---
+
+<div>
+  <button>Submit a Bug</button>
+</div>
```
Comment (10thfloor) [inline docs/content/bounties/responsible-disclosure.md:8]:
```suggestion
If you believe you may have found a security vulnerability in one of our products or platforms, send us an email: [security@dapperlabs.com](mailto:security@onflow.org?subject=Responsible%20disclosure)
```

**Pull request merged**